features/admin.groups.v1/pages/groups.tsx

```diff
@@ -187,41 +186,39 @@ const GroupsPage: FunctionComponent<any> = (): ReactElement => {
             value: ""
         };
 
-        if (isSuperOrganization() || isFirstLevelOrganization()) {
-            setUserStoresListRequestLoading(true);
-            getUserStoreList()
-                .then((response: AxiosResponse<UserstoreListResponseInterface[]>) => {
-                    if (storeOptions?.length === 0) {
-                        storeOptions.push(storeOption);
-                    }
-
-                    response.data.map((store: UserstoreListResponseInterface, index: number) => {
-                        if (store.name.toUpperCase() !== userstoresConfig.primaryUserstoreName) {
-                            setUserStoreRequestLoading(true);
-                            getAUserStore(store.id).then((response: UserStorePostData) => {
-                                const isDisabled: boolean = response.properties.find(
-                                    (property: UserStoreProperty) => property.name === "Disabled")?.value === "true";
-
-                                if (!isDisabled) {
-                                    storeOption = {
-                                        disabled: store.typeName === RemoteUserStoreManagerType.RemoteUserStoreManager,
-                                        key: index,
-                                        text: store.name,
-                                        value: store.name
-                                    };
-                                    storeOptions.push(storeOption);
-                                }
-                            }).finally(() => {
-                                setUserStoreRequestLoading(false);
-                            });
-                        }
-                    });
+        setUserStoresListRequestLoading(true);
+        getUserStoreList()
+            .then((response: AxiosResponse<UserstoreListResponseInterface[]>) => {
+                if (storeOptions?.length === 0) {
+                    storeOptions.push(storeOption);
+                }
 
-                    setUserStoresList(storeOptions);
-                }).finally(() => {
-                    setUserStoresListRequestLoading(false);
+                response.data.map((store: UserstoreListResponseInterface, index: number) => {
+                    if (store.name.toUpperCase() !== userstoresConfig.primaryUserstoreName) {
```

Comment:
We recently added the capability to dynamically specify the primary userstore for console app. Therefore, comparing the userstore name with the predefined hardcoded name would break that functionality here I guess.

cc: @DilshanSenarath

Reply:
Resolving as per the offline discussion since this is existing code

Reply:
Here, since we have retrieved the user store from `userstoresConfig.primaryUserstoreName`, it is acceptable because `userstoresConfig.primaryUserstoreName` will be assigned based on the updated primary domain name. https://github.com/wso2/identity-apps/blob/d1c5655228a95ff7a3fb7c20e9e842685158b998/features/admin.extensions.v1/configs/userstores.tsx#L22-L28